fare/file_management/education_level.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Thinking carefully about this I think the best option is to include the content of both files inside a JSON structure in the general config.py. This setting may be used also by other modules so I think it is not optimal to leave it as it is. I would just copy the two file contents in two vars FARE_EDUCATION_LEVEL and FARE_SUBJECT. What do you think? @CorsoGabriele